src/main/java/com/mycompany/exercisethree/AdvanceCalculator.java

```diff
@@ -0,0 +1,80 @@
+package com.mycompany.exercisethree;
+
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ *
+ * @author Cristian Gallego
+ */
+public class AdvanceCalculator extends Calculator {
+
+    @Override
+    public void operateCalculator() {
+        
+        Scanner getInt = new Scanner(System.in);
```

Comment:
This is not the right naming convection, use something like "optionSelected" or "option", etc

Reply:
Done.